rpc/src/rpc_service.rs

```diff
@@ -737,6 +743,7 @@ mod tests {
             RpcHealth::stub(),
             block_commitment_cache,
             None,
+            false,
```

Comment:
There are some compile errors here because the argument moved, you can typecheck the tests with `cargo check --test`.

Reply:
@ruuda I wonder if it's on the `prometheus-master` branch as well 
e.g. https://github.com/solana-labs/solana/pull/27665/files#diff-ae00f1b802baaa57eb71ae6bf8425ca1c4e8e32b3e82ef2f5a122ac2307cb01dR691

Reply:
Hmm, might be ... could you address this nonetheless?

We also have a `prometheus-1.13` branch that is compatible with Solana 1.13. It is more up to date because that is the one we actually use, maybe it is fixed there, but it doesn’t target the `master` branch.